.github/workflows/lint.yml

```diff
@@ -0,0 +1,47 @@
+name: Lint
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - '**.sol'
+      - '**.yml'
+  push:
+    branches:
+      - main
+    paths:
+      - '**.sol'
+      - '**.yml'
+
+env:
+  FOUNDRY_PROFILE: ci
+
+jobs:
+  tests:
```

Comment:
```suggestion
  lint:
```

<details><summary>How this is confusing</summary>

See workflow name in the top left but job name next to the green checkmark.

![image](https://github.com/cowprotocol/contracts/assets/58218759/b1fc8ade-9716-4221-a21e-b07e7055d5a3)
</details>